### PR TITLE
[Keystone] remove tempest setup in keystone helm-chart

### PR DESCRIPTION
**Pull request by rajivmucheli**
The new setup is located at https://github.com/sapcc/helm-charts/tree/master/openstack/tempest/keystone-tempest

### CODE REVIEW

**Comment by rajivmucheli** [comment]
This change was introduced before tempest tests were green. Let's take a step back, merge the previous tempest PR, validate it for few days and look into this PR.

**Review by stanislav-zaprudskiy — CHANGES_REQUESTED** [review]
Removing references to the deleted template files is also required in [`templates/configmap-etc.yaml`](https://github.com/sapcc/helm-charts/blob/remove_tempest_keystone/openstack/keystone/templates/configmap-etc.yaml) (e.g. `rally_deployment_config.json`)

**Comment by stanislav-zaprudskiy** [comment]
> @bbobrov shall we remove this spec https://github.com/sapcc/helm-charts/blob/remove_tempest_keystone/openstack/keystone/values.yaml#L190-L198 now that all PR's are merged and working ?

I believe it would be failing during deployments where `.Values.tempest` hash is not provided, e.g. while performing this check here https://github.com/sapcc/helm-charts/blob/remove_tempest_keystone/openstack/keystone/templates/etc/_policy.yaml.tpl#L24.

**Comment by rajivmucheli** [comment]
> I believe it would be failing during deployments where `.Values.tempest` hash is not provided, e.g. while performing this check here https://github.com/sapcc/helm-charts/blob/remove_tempest_keystone/openstack/keystone/templates/etc/_policy.yaml.tpl#L24.

When i ran it locally it worked and the idea is to control this in secrets (tempest is only enabled in qa).

**Review by stanislav-zaprudskiy — APPROVED** [review]

File: openstack/keystone/values.yaml [hunk 1]
```diff
@@ -189,14 +189,6 @@ cache:
...
 tempest:
   enabled: false
-  image: "rally-openstack"
-  #imageTag: "ccloud-1.5.0"
-  imageTag: ""
-  adminPassword: ""
-  userPassword: ""
-  # tempest domain-id  -- do not set in production environments!
-  domainId: not_set_this_in_production
-  adminProjectId: not_set_in_production
```
Comment (bbobrov) [inline openstack/keystone/values.yaml:199]: This value is still needed

File: openstack/keystone/values.yaml [hunk 2]
```diff
@@ -189,14 +189,6 @@ cache:
 
 tempest:
   enabled: false
-  image: "rally-openstack"
-  #imageTag: "ccloud-1.5.0"
-  imageTag: ""
-  adminPassword: ""
-  userPassword: ""
-  # tempest domain-id  -- do not set in production environments!
-  domainId: not_set_this_in_production
```
Comment (bbobrov) [inline openstack/keystone/values.yaml:198]: This value is still needed

**Review by bbobrov — CHANGES_REQUESTED** [review]
This must also have an approval from @misamoylov

File: openstack/keystone/values.yaml (hunk 1 above)
Comment (rajivmucheli) [reply openstack/keystone/values.yaml:199]: this value is enabled in secrets.

File: openstack/keystone/values.yaml (hunk 2 above)
Comment (rajivmucheli) [reply openstack/keystone/values.yaml:198]: this value is enabled in secrets.

**Review by bbobrov — COMMENTED** [review]
This is good, but it still must be approved by @misamoylov

**Pull request merged**